community/kernel/src/main/java/org/neo4j/graphdb/GraphDatabaseService.java

```diff
@@ -198,10 +187,9 @@
     boolean isAvailable( long timeout );
```

Comment:
Actually, no - you're right. It should be moved to `GraphDatabase`, rather than `GraphDatabaseService`.